src/main/scala/atom/Atom.scala

```diff
@@ -77,12 +81,20 @@ trait Notification extends js.Object {
   def dismiss(): Unit = js.native
 }
 
+@JSExportAll
```

Comment:
I'm not sure I understand this `@JSExportAll` annotation here and in other places. `NotificationButton` is not going to be used in the JS code, so isn't it enough just to make it extend `JSObject`?

Reply:
See [nodejs/child_process/ExecOptions.scala](https://github.com/scalajs-io/nodejs/blob/master/app/common/src/main/scala/io/scalajs/nodejs/child_process/ExecOptions.scala) for example